scripts/linters/test_files/invalid_prod_validation_jobs_one_off.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Python file with invalid syntax, used by scripts/linters/
+python_linter_test. This file is to use with
```

Comment:
use --> be used

Maybe explain what the invalid syntax is? I can't tell from a quick glance.

Reply:
This file is not needed anymore.